indico/modules/events/registration/fields/base.py

```diff
@@ -14,7 +14,12 @@
 from indico.util.marshmallow import not_empty
 
 
-class BillableFieldDataSchema(mm.Schema):
+class FieldSetupSchemaBase(mm.Schema):
```

Comment:
Just so we don't forget it: `RegistrationFormCloner`'s `_clone_form_items` method will need to map the field ids. Probably this needs a second loop over the items after all the fields have been cloned because the referenced field may not have been cloned yet when the conditional one is being cloned.